### DISCUSSION TITLE
Change default order of collection in control panel

### DISCUSSION THREAD

**Discussion by richardhj** [statamic/cms#7624] [Q&A] [answered]
How can I change the default sorting order of collections in the control panel?

I am setting up a news collection and want them to be sorted by date descending, per default, for all users.

Duplicate of: https://statamic.com/forum/2477-change-sort-order-of-collection-items-in

**Comment by jasonvarga** [comment, marked as answer]
Edit the collection and flip the sort direction.

![CleanShot 2023-02-28 at 10 27 09](https://user-images.githubusercontent.com/105211/221899416-380087c2-abe2-434a-b7d6-858b833dba7e.png)

**Comment by richardhj** [comment]
This works - thanks! But how does Statamic know to order by the Date field and not by the Title field? 
I checked with other collections, and I cannot understand how Statmic chooses the order field - I expect some magic to happen here 😄

**Reply by jasonvarga** [reply]
Yes there is a little cleverness built in.

It goes like this:
- If you've set a custom `sort_by` field to your collection yaml file, it'll use that field.
- If it's orderable, it'll use `order`.
- If it's dated, it'll use `date`.
- Otherwise, it'll use `title`.

**Reply by richardhj** [reply]
Thanks a lot for clarification!
